dockerw

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+source .env
+
+RUN_MODE=run
+BUILD_MODE=build
+UP_MODE=up
+
+print_usage() {
+  echo "Usage: $0 build|run -i| -v|--version=[VERSION] -i|--image=[IMAGE]  -c|--container=[CONTAINER_NAME] "
+  echo ""
+  echo "build - simply build image without running"
+  echo "run   - runs image and attempts to pull it if it cannot be found locally"
+  echo "up    - build and run image locally"
+  echo "[IMAGE] = Image Name. Default='$IMAGE'"
+  echo "[VERSION] = Image version. Default='$VERSION'"
+  echo "[CONTAINER_NAME] = Running Container Name. Default='$CONTAINER_NAME'"
+}
+
+print_wrong_mode() {
+  echo "Mode not correctly specified. Modes are: $BUILD_MODE|$RUN_MODE|$UP_MODE"
+  exit 2
+}
+
+print_info() {
+  echo "----------------------------------------"
+  echo "Running in '$MODE' mode"
+  echo "VERSION           = ${VERSION}"
+  echo "IMAGE             = ${IMAGE}"
+  echo "CONTAINER_NAME    = ${CONTAINER_NAME}"
+  echo "----------------------------------------"
+}
+
+parse_args() {
+  for i in "$@"; do
+    case $i in
+    build)
+      MODE="$BUILD_MODE"
+      shift
+      ;;
+    run)
+      MODE="$RUN_MODE"
+      shift
+      ;;
+    up)
+      MODE="$UP_MODE"
+      shift
+      ;;
+    -i=* | --image=*)
+      IMAGE="${i#*=}"
+      shift
+      ;;
+    -v=* | --version=*)
+      VERSION="${i#*=}"
+      shift
+      ;;
+    -c=* | --container=*)
+      CONTAINER_NAME="${i#*=}"
+      shift
+      ;;
+    *) ;; # unknown option
+    esac
+  done
+}
+
+docker_build() {
+  docker build . -t $IMAGE:$VERSION
+}
+
+docker_run() {
+  if [ ! -f js/config.js ]; then
+    echo "Creating empty config.js file"
+    touch js/config.js
+  fi
+
+  echo "Stopping container $CONTAINER_NAME"
+  docker stop $CONTAINER_NAME
+  docker rm $CONTAINER_NAME
+
+  echo "Starting container $CONTAINER_NAME"
+  docker run -p80:80 -v ${PWD}/js:/usr/share/nginx/html/js --name=$CONTAINER_NAME -d $IMAGE:$VERSION
```

Comment:
Maybe don't mount this dir in the future? Just really need to mount in the config file